v1/plugin/flags.go

```diff
@@ -0,0 +1,56 @@
+package plugin
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/urfave/cli"
+)
+
+var (
+	flConfig = cli.StringFlag{
+		Name:  "config",
+		Usage: "config file to use",
+	}
+	// If no port was provided we let the OS select a port for us.
+	// This is safe as address is returned in the Response and keep
+	// alive prevents unattended plugins.
+	flPort = cli.StringFlag{
+		Name:  "port",
+		Usage: "port to listen on",
+	}
+	flLogLevel = cli.IntFlag{
+		Name:  "log-level",
+		Usage: "log level - 0:panic 1:fatal 2:error 3:warn 4:info 5:debug",
+		Value: 2,
+	}
+	flPprof = cli.BoolFlag{
+		Name:  "pprof",
+		Usage: "set pprof",
+	}
+	flTLS = cli.BoolFlag{
+		Name:  "tls",
+		Usage: "enable TLS",
+	}
```

Comment:
Please check if this flag works as expected. Below there is description of the current behaviour:

a) set --tls flag to enable TLS
```
$ ./snap-plugin-collector-rand  --stand-alone --port 8184 --tls
```
But `TLSEnabled` is still false

```
 Received: {"Meta":{"Type":0,"Name":"test-rand-collector","Version":1,"RPCType":2,"RPCVersion":1,"ConcurrencyCount":5,"Exclusive":false,"Unsecure":true,"CacheTTL":0,"RoutingStrategy":0,"CertPath":"","KeyPath":"","TLSEnabled":false,"RootCertPaths":""},"ListenAddress":"127.0.0.1:8184","PprofAddress":"0","Type":0,"State":0,"ErrorMessage":""}
```